.travis.yml

```
@@ -1,19 +1,11 @@
 language: generic
 sudo: false
 
-# notifications:
-#   webhooks:
-#     urls:
-#       - https://discordapp.com/api/webhooks/475124635164213248/HnIVHWdsJqGjNZdyKynaFy1oDEoxoMb50gGDubBsffiHmkMp_mjQXFyrYL0wUgdxOErY
-#     on_success: always  # options: [always|never|change] default: always
-#     on_failure: always  # options: [always|never|change] default: always
-#     on_start: never     # options: [always|never|change] default: always
-
 env:
-    global:
-    - BYOND_MAJOR="512"
-    - BYOND_MINOR="1452"
-    - DM_BUILDFILE="baystation12.dme"
+  global:
+    BYOND_MAJOR="512"
+    BYOND_MINOR="1467"
```

```suggestion
    BYOND_MINOR="1463"
```

Должна стоять та же версия, что и на хосте, сейчас там вроде 512.1463.

---

Выставил верную версию в 63255ca